### PR TITLE
Yinglong infer

### PR DESCRIPTION
**Pull request by zhiminzhang0830**
### PR types

Others
### PR changes

Others
### Describe

add visiualize
add 24 layers
adopt to paddle 2.6

### CODE REVIEW

File: examples/yinglong/doc/fig_arch1.jpg
Comment (HydrogenSulfate) [inline examples/yinglong/doc/fig_arch1.jpg:1]: 图片上传云端

File: examples/yinglong/doc/result.gif
Comment (HydrogenSulfate) [inline examples/yinglong/doc/result.gif:1]: 图片上传云端

File: examples/yinglong/plot.py [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+    Args:
+        foldername (str): Output folder name.
+        data_dict (Dict[str, Union[np.ndarray, paddle.Tensor]]): Data in dict.
+        visu_keys (Tuple[str, ...]): Keys for visualizing data. such as ("output_6h", "target_6h").
+        xticks (Tuple[float, ...]): The list of xtick locations.
+        xticklabels (Tuple[str, ...]): The x-axis' tick labels.
+        yticks (Tuple[float, ...]): The list of ytick locations,
+        yticklabels (Tuple[str, ...]): The y-axis' tick labels.
+        vmin (float): Minimum value that the colormap covers.
```
Comment (HydrogenSulfate) [inline examples/yinglong/plot.py:184]: minimum --> minimal

File: examples/yinglong/plot.py [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+        filename (str): Output file name.
+        pred (np.ndarray): The predict data.
+        target (np.ndarray): The target data.
+        pred_key (str): The key of predict data.
+        target_key (str): The key of target data.
+        xticks (Tuple[float, ...]): The list of xtick locations.
+        xticklabels (Tuple[str, ...]): The x-axis' tick labels.
+        yticks (Tuple[float, ...]): The list of ytick locations.
+        yticklabels (Tuple[str, ...]): The y-axis' tick labels.
+        vmin (float): Minimum value that the colormap covers.
```
Comment (HydrogenSulfate) [inline examples/yinglong/plot.py:88]: Minimal

**Review by HydrogenSulfate — CHANGES_REQUESTED** [review]

**Comment by zhiminzhang0830** [comment]
@HydrogenSulfate done

**Review by HydrogenSulfate — APPROVED** [review]
LGTM

**Pull request merged**